content/components/action-menu.mdx

```diff
@@ -102,89 +128,235 @@ Action menu is used for actions, navigation, to display secondary options, or si
+    />
+    <Caption>Don't add additional form elements inside a menu.</Caption>
+  </Dont>
+</DoDontContainer>
+
+Since you can't include form elements within a menu that also means that you can't combine a [select panel](/components/selectpanel) with a action menu.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234094256-abf3f1ad-2d90-48f9-84aa-a8e17cc64c47.png"
+    />
+    <Caption>If you want to present filterable data do this through a dialog.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/234094263-399afdd0-b2e5-45dc-a218-db4a7dcdd9be.png"
+    />
+    <Caption>Don't mix select panels with action menus.</Caption>
+  </Dont>
+</DoDontContainer>
+
+When using submenus don't use the trailing visual to communicate selected values within their submenu.
+Leave this space for submenu indicators.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232562573-a687110d-06cb-4756-8fb3-6b7e05665dd4.png"
+    />
+    <Caption>Use trailing visuals for submenu indicators or keyboard shortcuts.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/232562567-bb96a0cb-a796-4834-bf91-e3ddff86c312.png"
+    />
+    <Caption>Don't use trailing visual to communicate selected items in submenus.</Caption>
+  </Dont>
+</DoDontContainer>
+
+On smaller viewports the menu should always be repositioned to fit within the viewport. This could mean that a submenu overlaps it's parent menu.
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/233361999-3b5ab994-152d-455e-b9bc-a9170a1951b1.png"
+    />
+    <Caption>Reposition menus to stay within the viewport, especially on narrow viewports.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/980622/233361993-cf3e0b21-e584-4780-a0e1-42f03fc4d395.png"
+    />
+    <Caption>Don't position menus off screen, especially on narrow viewports.</Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
+
+<img
+  src="https://user-images.githubusercontent.com/18661030/226486461-c43f75c5-d9ed-415f-907c-9d2dbf71e3a2.png"
+  width="960"
+  alt=""
+/>
+<Caption>
+  The trigger button can be a standard or icon button. For single select menus that display the selection in the
+  trigger, a label must be persist either internally or externally.
+</Caption>
+
+<img
+  src="https://user-images.githubusercontent.com/980622/234111244-aa3e9e6c-5ec4-4dbb-b0e9-fc11e97f6eba.png"
+  width="960"
+  alt=""
+/>
+<Caption>Menus can contain simple actions, links, single select items, multi select items or toggles.</Caption>
```

Comment:
What do you mean by toggles here?

Reply:
They are individual selections that behave like toggles. For example in Safari when you click the developer menu open you can disable images. This behaves as a toggle.

<img width="527" alt="Screenshot 2023-04-25 at 10 26 41" src="https://user-images.githubusercontent.com/980622/234234480-59869d19-c6b8-4020-8fce-5dd2aab23397.png">